configs/Kconfig

```diff
@@ -98,4 +106,4 @@ config DEMO_ANIMATION
     bool "Build animation demo"
     default y
     depends on DEMO_APPLICATIONS 
-endmenu
+endmenu
```

Comment:
Be aware of [a new line to the end of a file?](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file).